packages/workflow/test/TelemetryHelpers.test.ts

```diff
@@ -780,6 +780,392 @@ describe('generateNodesGraph', () => {
 
 		expect(() => generateNodesGraph(workflow, nodeTypes)).not.toThrow();
 	});
+
+	test.only('should add run and items count', () => {
+		const workflow: IWorkflowBase = {
```

Comment:
Should we move the JSON to a helper? The helper will create this whole json and override any parameter provided by default. Same for RunData. Will make much easier to read the tests as half of the file is just JSON